index.html

```diff
@@ -8,9 +8,11 @@
   <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet">
   <link href="css/bootstrap.min.css" rel="stylesheet">
   <link href="css/custom.css" rel="stylesheet">
+
   <link rel="stylesheet" href="css/gov-uk-styles.css">
-  
+  <link rel="stylesheet" href="css/move-remove-buttons.css"> 
   <title>Summary builder</title>
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>

```

Comment:
I'm an old fogey and was stubbonly against jquery, but there's no real harm in incuding it